test/tags/definition.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace A\B;
+//        ^ definition.module
+//          ^ definition.module
+
+interface T {
+//        ^ definition.interface
+}
+
+trait U {
+//    ^ definition.interface
+}
+
+function f() {
+         ^ definition.function
```

Comment:
this needs to be commented:
```suggestion
function f() {
//       ^ definition.function
```